### PR TITLE
add about page

### PR DESCRIPTION
**Pull request by lukaselmer**

### CODE REVIEW

**Comment by lukaselmer** [comment]
please review and merge #3 first

**Review by schmijos — APPROVED** [review]

File: web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -29,7 +29,13 @@
         <%= render @view_module, @view_template, assigns %>
       </main>
 
-    </div> <!-- /container -->
+    </div>
+    
+    <footer class="footer">
+      <div class="container">
+        <p class="text-muted"><a href="/">Home</a> <a href="/about">About</a></p>
```
Comment (coorasse) [inline web/templates/layout/app.html.eex]: can you use path helpers? like `about_path`?

File: web/static/css/app.css [hunk 2]
```diff
@@ -1 +1,10 @@
-/* This file is for your main application css. */
\ No newline at end of file
+.footer {
+  background: linear-gradient(to bottom, #66afe9, #2e6da4);
+  height: 60px;
+  padding-top: 20px;
+}
+
+.footer a {
```
Comment (coorasse) [inline web/static/css/app.css:7]: I ❤️ scss

**Review by coorasse — APPROVED** [review]

File: web/templates/layout/app.html.eex (hunk 1 above)
Comment (lukaselmer) [reply web/templates/layout/app.html.eex]: i'll try

**Pull request merged**